src/screens/FeedScreen/ContentPoll.js

```diff
@@ -170,14 +171,14 @@ const stylesComponent = (pollLength) =>
     feedDate: {
       fontFamily: fonts.inter[400],
       fontSize: normalizeFontSize(12),
-      color: COLORS.black,
+      color: colors.black,
       lineHeight: 18
     },
     point: {
       width: 4,
       height: 4,
       borderRadius: 4,
-      backgroundColor: COLORS.gray8,
+      backgroundColor: colors.gray,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated from `COLORS.gray8` to `colors.gray`. Make sure that `colors.gray` is defined and is the intended color.
```diff
- backgroundColor: COLORS.gray8
+ backgroundColor: colors.gray
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->